activerecord/lib/active_record/nested_attributes.rb

```diff
@@ -165,6 +165,9 @@ class TooManyRecords < ActiveRecordError
     #
     #   member.posts.first.title # => '[UPDATED] An, as of yet, undisclosed awesome Ruby documentation browser!'
     #   member.posts.second.title # => '[UPDATED] other post'
+    #  
+    # However, this only applies if the parent model is being updated as well. For example, If you wanted to create a
+    # +member+ named _joe_ and wanted to update the +posts+ at the same time, that would give an error.
```

Comment:
Thanks for the PR! 

Can you note what that error is so users know they aren't hitting some other problem and assume it's this caveat?